src/data-handling/Trip.js

```diff
@@ -1 +1,70 @@
-export default class Trip {};
\ No newline at end of file
+import moment from 'moment';
+moment().format();
+
+export default class Trip {
```

Comment:
Exporting on declare makes sense for class prototypes, considering they won't be called in that file. Good job!